crates/uv-python/src/downloads.rs

```diff
@@ -7,8 +9,11 @@ use std::task::{Context, Poll};
 use std::time::{Duration, SystemTime};
 
 use futures::TryStreamExt;
+use itertools::Itertools;
+use once_cell::sync::OnceCell;
```

Comment:
Should we use `std::cell::OnceCell` instead?

Reply:
I wanted to, but [get_or_try_init](https://doc.rust-lang.org/std/cell/struct.OnceCell.html#method.get_or_try_init) is nightly-only